### PR TITLE
Fix bias adjustment calcs for GDPS and GFS

### PR DESCRIPTION
**Pull request by dgboss**
GFS and GDPS weather models only contain predictions at 3 hour intervals and no prediction exists at 20:00 UTC. As a result, there is no 20:00 hour when training a linear regression model and using it to make a prediction.

As a work around I am taking the prediction at 18:00 and getting a bias adjusted value, do the same at 21:00 and finally use linear interpolation to estimate the bias adjusted value at 20:00.
# Test Links:
[Landing Page](https://wps-pr-3308.apps.silver.devops.gov.bc.ca/)
[MoreCast 2.0](https://wps-pr-3308.apps.silver.devops.gov.bc.ca/morecast-2)
[Percentile Calculator](https://wps-pr-3308.apps.silver.devops.gov.bc.ca/percentile-calculator)
[MoreCast](https://wps-pr-3308.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-3308.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-3308.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-3308.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-3308.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-3308.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

File: api/app/jobs/common_model_fetchers.py [hunk 1]
```diff
@@ -373,9 +433,10 @@ def _process_model_run_for_station(self,
                 noon_prediction = construct_interpolated_noon_prediction(
```
Comment (brettedw) [inline api/app/jobs/common_model_fetchers.py:433]: Was this part of the code was meant to handle interpolating the noon predictions?

Comment (dgboss) [reply api/app/jobs/common_model_fetchers.py:433]: Yes, it constructs the 'prediction' part for GFS and GDPS properly as far as I can tell. Then I had to tweak the part for constructing the bias corrected values.

**Review by brettedw — APPROVED** [review]
👌

**Review by conbrad — APPROVED** [review]
Nice

**Pull request merged**